src/darwin/Framework/CHIP/templates/availability.yaml

```diff
@@ -7705,6 +7704,13 @@
           GeneralDiagnostics:
               # Not in the spec yet.
               - AverageWearCount
+              # New things, not quite finalized.
```

Comment:
These should (1) be under the "Future" release, not here, and (2) should be commented "# Targeting Spring 2024 Matter release" like the other bits down there.